src/types.ts

```diff
@@ -23,3 +23,46 @@ export type Permission =
   | "collection:create"
   | "group:create"
   | "record:create";
+
+interface User {
+  id: string;
+  principals: string[];
+  bucket: string;
+}
+
+interface ServerCapability {
+  description: string;
+  url: string;
```

Comment:
```suggestion
  url: string;
  version: string;
```

Reply:
Based on the response from `https://kinto.dev.mozaws.net/v1/`, not all capabilities have the `version` property. What we could do, though, is define it as optional, that way if someone does a check like `if (capability.version)` the TypeScript compiler will know it's a `string` without the user needing to cast it to one.

Reply:
Yes I would say that version is optional, it is part of the API. The only one that don't have a version property are plugins that ships with kinto itself.

Reply:
Ah, gotcha! In that case I think it makes sense to have it defined. Since it's an optional field, users will need to check that it's defined before accessing it, but they won't need to check that it's a string.